machineconfiguration/v1alpha1/0000_80_machineconfigstate.crd.yaml

```diff
@@ -0,0 +1,185 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: machineconfigstates.machineconfiguration.openshift.io
+  labels:
+    "openshift.io/operator-managed": ""
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  # group name to use for REST API: /apis/<group>/<version>
+  group: machineconfiguration.openshift.io
+  scope: Cluster
+  names:
+    kind: MachineConfigState
+    singular: machineconfigstate
+    plural: machineconfigstates
+    shortNames:
+      - mcs
+  versions:
+    - name: v1alpha1
+      # Each version can be enabled/disabled by Served flag.
+      served: true
+      # One and only one version must be marked as the storage version.
+      storage: true
+      subresources:
+        status: {}
+      additionalPrinterColumns:
+        - jsonPath: .status.health
+          name: Health
+          type: string
+        - jsonPath: .status.conditions[0].state
+          name: Progress
+          type: string
+        - jsonPath: .status.conditions[0].reason
+          name: Reason
+          type: string
+        - jsonPath: .status.conditions[0].phase
+          name: Phase
+          type: string
```

Comment:
Relying on ordering of conditions for printer columns is probably going to be quite fragile. What do you actually want to display here? You can use a json query to select a particular condition based on its type field, that's what we typically do